api/nodes/serializers.py

```diff
@@ -560,8 +560,7 @@ def get_node_count(self, obj):
                   FROM parents JOIN osf_noderelation ON parents.PARENT_ID = osf_noderelation.child_id
                   WHERE osf_noderelation.is_node_link IS FALSE
                 ), has_admin AS (SELECT * FROM osf_contributor WHERE (node_id IN (SELECT parent_id FROM parents) OR node_id = %s) AND user_id = %s AND admin IS TRUE LIMIT 1)
```

Comment:
The WITH RECURSIVE portion of the query above is actually looking up the tree to see if you have admin perms on any parents, grandparents, etc., which would enable you to see children of the current node downstream.

This is not returning multiple levels of descendants, just the next level, which is correct behavior, and matches what NodeChildrenList returns.